sql/task2/electivesScript.sql

```diff
@@ -0,0 +1,17 @@
+CREATE DATABASE IF NOT EXISTS test;
+
```

Comment:
You may want to add `use` sql statement for the db, so that when the script is executed the following commands are executed against the correct database.